replicated-security/noble-1/noble-1-genesis-without-ccv.json

```
@@ -0,0 +1,233 @@
+          "default_send_enabled": true
+        },
+        "balances": [],
+        "supply": [],
+        "denom_metadata": [
+          {
+            "base": "uusdc",
+            "denom_units": [
+              {
+                "aliases": ["microusdc"],
+                "denom": "uusdc",
+                "exponent": 0
+              },
+              {
+                "aliases": ["milliusdc"],
+                "denom": "musdc",
+                "exponent": 3
+              },
+              {
+                "aliases": [],
+                "denom": "usdc",
+                "exponent": 6
+              }
+            ],
+            "description": "",
+            "display": "usdc",
+            "name": "USDC",
+            "symbol": "USDC"
+          }
+        ]
+      },
+      "capability": {
+        "index": "1",
+        "owners": []
+      },
+      "ccvconsumer": {},
+      "crisis": {
+        "constant_fee": {
+          "denom": "stake",
+          "amount": "1000"
+        }
+      },
+      "evidence": {
+        "evidence": []
+      },
+      "feegrant": {
+        "allowances": []
+      },
+      "ibc": {
+        "client_genesis": {
+          "clients": [],
+          "clients_consensus": [],
+          "clients_metadata": [],
+          "params": {
+            "allowed_clients": ["06-solomachine", "07-tendermint"]
+          },
+          "create_localhost": false,
+          "next_client_sequence": "0"
+        },
+        "connection_genesis": {
+          "connections": [],
+          "client_connection_paths": [],
+          "next_connection_sequence": "0",
+          "params": {
+            "max_expected_time_per_block": "30000000000"
+          }
+        },
+        "channel_genesis": {
+          "channels": [],
+          "acknowledgements": [],
+          "commitments": [],
+          "receipts": [],
+          "send_sequences": [],
+          "recv_sequences": [],
+          "ack_sequences": [],
+          "next_channel_sequence": "0"
+        }
+      },
+      "interchainaccounts": {
+        "controller_genesis_state": {
+          "active_channels": [],
+          "interchain_accounts": [],
+          "ports": [],
+          "params": {
+            "controller_enabled": true
+          }
+        },
+        "host_genesis_state": {
+          "active_channels": [],
+          "interchain_accounts": [],
+          "port": "icahost",
+          "params": {
+            "host_enabled": true,
+            "allow_messages": []
+          }
+        }
+      },
+      "packetfowardmiddleware": {
+        "params": {
+          "fee_percentage": "0.000000000000000000"
+        },
+        "in_flight_packets": {}
+      },
+      "params": {
+        "params": {
+          "authority": "noble1lwklzs6ejglapp6eccxtfmajwykt8nrdh8fjp8"
+        }
+      },
+      "slashing": {
+        "params": {
+          "signed_blocks_window": "100",
+          "min_signed_per_window": "0.500000000000000000",
```

```suggestion
          "signed_blocks_window": "10000",
          "min_signed_per_window": "0.500000000000000000",
```
If we expect a block time of 5s, setting the window to 10,000 blocks would give validators almost 7 hours to join (as opposed to 4 minutes with a 100 block window).